### PR TITLE
Add descheduler repo

### PR DESCRIPTION
**Pull request by stevehipwell**

### CODE REVIEW

**Comment by mattfarina** [comment]
The [Artifact Hub](https://artifacthub.io/), a CNCF sandbox project, is another place you can list your repos. To do that, you login (or create an account), create an organization, and list your repository. You can link it to a user or organization.

File: repos.yaml [hunk 1]
```diff
@@ -1543,5 +1543,10 @@ repositories:
...
     maintainers:
-      - name: Leo Vigna 
-        email: leo@vulcan.link 
+      - name: Leo Vigna
+        email: leo@vulcan.link
+  - name: descheduler
+    url: https://kubernetes-sigs.github.io/descheduler/
+    maintainers:
+      - name: Kubernetes SIG Scheduling
+        email: kubernetes-sig-scheduling@googlegroups.com
```
Comment (mattfarina) [inline repos.yaml:1552]: @stevehipwell I would love to include descheduler in the Helm Hub. But, I don't see you listed as a chair of sig-scheduling nor a maintainer on descheduler. I would like to have one of the owners verify they are ok being listed as a maintainer prior to merging.

Comment (damemi) [reply repos.yaml:1552]: I'm a descheduler maintainer, gave my +1 above :)

@seanmalloy and @ingvagabund can also add their approvals if you'd like

Comment (mattfarina) [reply repos.yaml:1552]: @damemi Somehow I missed that. Sorry about that. My bad.

Comment (seanmalloy) [reply repos.yaml:1547]: I'm good with this.

**Pull request merged**